packages/webview_flutter/lib/platform_interface.dart

```diff
@@ -37,6 +48,67 @@ abstract class WebViewPlatform {
   int get id;
 }
 
+/// Settings for configuring a WebViewPlatform.
+///
+/// Initial settings are passed as part of [CreationParams], settings updates are sent with
+/// [WebViewPlatform#updateSettings].
+class WebSettings {
+  WebSettings({
+    this.javascriptMode,
+    this.hasNavigationDelegate,
+    this.debuggingEnabled,
+  });
+
+  final JavascriptMode javascriptMode;
+
+  /// Whether a [NavigationDelegate] should be used for this webview.
+  final bool hasNavigationDelegate;
+
+  /// Whether to enable the platform's webview content debugging tools.
+  ///
+  /// See also: [WebView.debuggingEnabled].
+  final bool debuggingEnabled;
```

Comment:
nit: isDebuggingEnabled

Reply:
I think it makes sense to keep it consistent with the `WebView` parameter name, if we want to change both it will be a breaking change at this point.

Note that as far as I know the Flutter style guide doesn't prefer an "is" prefix for flags.